sympy/physics/mechanics/functions.py

```diff
@@ -413,6 +413,26 @@ def find_dynamicsymbols(expression, exclude=None):
 
     >>> find_dynamicsymbols(expr, [x, y])
     {Derivative(x(t), t)}
+
+    Optional Parameter
```

Comment:
@moorepants I changed it to Parameters. I had a doubt, that in my docstring for find_dynamicsymbols, do I need to specify that when I am considering "expression" to be a matrix and when as a vector.This is so because my code is failing checks with the exception mentioned:


 File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/sympy-1.1.2.dev0-py3.6.egg/sympy/physics/mechanics/functions.py", line 448, in find_dynamicsymbols

    expr=expression.to_matrix(reference_frame)

  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/sympy-1.1.2.dev0-py3.6.egg/sympy/matrices/matrices.py", line 1852, in __getattr__

    "%s has no attribute %s." % (self.__class__.__name__, attr))

AttributeError: MutableDenseMatrix has no attribute to_matrix.

Although I am intended to employ "to_matrix" on a vector only not on a MutableDenseMatrix.

Reply:
@moorepants and @gxyd please guide over this issue.